src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/What-Language.cshtml.cs

```diff
@@ -71,26 +71,27 @@ protected override void OnGetWithError()
     protected override void OnGetWithModel()
     {
         // we've redirected to self with user input and no errors, so javascript must be disabled
-        if (ServiceModel?.UserInput != null)
+        
+        // redirectingToSelf is only set when adding a new field
+        if(ServiceModel?.UserInput != null && Request.Query.TryGetValue("redirectingToSelf", out var redirectToSelf) && redirectToSelf == "true")
```

Comment:
Minor nitpick, can we use `is not null` instead of `!=` - guarantees that should there ever be any overrides to the equality operators in `UserInput` then it won't change the behaviour of this null check